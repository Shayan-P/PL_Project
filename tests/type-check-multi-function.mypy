checked

def two(a=10, b=30):
    print(a);
    print(b);
;

def zero():
    print(88);
;

two(1, 2);
zero();

