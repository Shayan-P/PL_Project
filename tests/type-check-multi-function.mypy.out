type-checked!
1
2
88
'finished
